templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{%  extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="background-color: red">Invalid credantials</p>
+  {% endif %}
+  <form action="{% url 'login' %}" method="post">

```

Comment:
Code style issue: The `action` attribute in the form tag is not following the correct naming convention for URL names. According to the checklist, you should use `-` to split words in URL identification parameter `name`, not `_`. Please adjust the URL name from `'login'` to `'login-view'` or similar.